resources/assets/js/components/AssessmentPlan/RatingGuideAnswer.tsx

```diff
@@ -102,7 +136,9 @@ const RatingGuideAnswer: React.FunctionComponent<
           onChange={(event): void =>
```

Comment:
This is probably safe to leave for a future refactor, but we should stick to using named functions instead of anonymous ones (especially inside components that are repeated multiple times).

Reply:
For performance purposes?

Reply:
I'll keep that in mind in the future